### ISSUE TITLE
[Failing Test] capa-conformance-stable-k8s-master (ci-cluster-api-provider-aws-make-conformance-stable-k8s-ci-artifacts)

### ISSUE THREAD

**Issue by droslean** [kubernetes/kubernetes#85335]
**Which jobs are failing**:
`capa-conformance-stable-k8s-master (ci-cluster-api-provider-aws-make-conformance-stable-k8s-ci-artifacts)`

**Since when has it been failing**:
`13th Nov 15:54 PST`

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-informing#capa-conformance-stable-k8s-master

**Reason for failure**:
```console
 I1115 06:25:44.925100   14412 clusterclient.go:242] Waiting to acquire client... server not yet available: Get https://test1-apiserver-58941778.us-west-2.elb.amazonaws.com:6443/api?timeout=32s: EOF
I1115 06:25:54.927537   14412 clusterclient.go:242] Waiting to acquire client... server not yet available: Get https://test1-apiserver-58941778.us-west-2.elb.amazonaws.com:6443/api?timeout=32s: EOF
I1115 06:26:04.926351   14412 clusterclient.go:242] Waiting to acquire client... server not yet available: Get https://test1-apiserver-58941778.us-west-2.elb.amazonaws.com:6443/api?timeout=32s: EOF
I1115 06:26:14.941178   14412 clusterclient.go:242] Waiting to acquire client... server not yet available: Get https://test1-apiserver-58941778.us-west-2.elb.amazonaws.com:6443/api?timeout=32s: EOF
I1115 06:26:15.067574   14412 clusterclient.go:242] Waiting to acquire client... server not yet available: Get https://test1-apiserver-58941778.us-west-2.elb.amazonaws.com:6443/api?timeout=32s: EOF
F1115 06:26:15.067791   14412 alpha_phase_apply_addons.go:51] unable to create cluster client: failed to acquire new client: timed out waiting for the condition
make: *** [Makefile:327: create-cluster-management] Error 1
+ exit-handler
+ dump-logs
+ echo '=== versions ==='
=== versions ===
+ timeout 600 kubectl --kubeconfig=/root/.kube/kind-config-clusterapi delete cluster test1
cluster.cluster.x-k8s.io "test1" deleted
+ true
++ kind get kubeconfig-path --name=clusterapi
+ timeout 600 kubectl --kubeconfig=/root/.kube/kind-config-clusterapi wait --for=delete cluster/test1
error: timed out waiting for the condition on clusters/test1 
```

**Anything else we need to know**:
/cc @alenkacz @hasheddan @alejandrox1 
/priority critical-urgent
/milestone v1.17
/sig cluster-lifecycle

**Comment by dims** [comment]
https://prow.k8s.io/?job=ci-cluster-api-provider-aws-make-conformance-stable-k8s-ci-artifacts seems to have recovered, though https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-aws#capa-conformance-stable-k8s-master still has stale data

**Comment by liggitt** [comment]
test grid is showing green again now, fix in #85351 worked

/close
